### DISCUSSION TITLE
Cant use my own services

### DISCUSSION THREAD

**Discussion by SucculentGreen** [rustdesk/rustdesk#4040] [General]
Everytime I try and run the rust desk exe it creates new services that just dont work......

when i create my own hbbr and hbbs it doesnt recognise them at all and keeps making new ones over and over

**Comment by rustdesk** [comment]
https://www.reddit.com/r/rustdesk/comments/12g9jmi/installing_your_own_rustdesk_server_on_a_windows/

**Reply by SucculentGreen** [reply]
but the nssm setup worked perfectly why must i use pm2 now?

**Reply by SucculentGreen** [reply]
All of this worked perfectly 2 weeks ago.....

but now I have followed my literal step by step guide I made during the first build and it keeps creating new hbbs and hbbr services.
its as though there is a script running that forces this on me...

please any help would be amazing?
